src/Features/Core/Portable/IntroduceVariable/AbstractIntroduceVariableService.State.cs

```diff
@@ -198,6 +198,12 @@ private TExpressionSyntax GetExpressionUnderSpan(SyntaxTree tree, TextSpan textS
                     // We are pointing in the trailing whitespace trivia of a token, we shouldn't include that token
                     startToken = startToken.GetNextToken();
                     var newStart = startToken.Span.Start;
+
+                    if (textSpan.End <= newStart)
+                    {
+                        return null;
```

Comment:
:bulb: Explanation of the scenario would be helpful

> Additional trivia exists between 'textSpan' and the start of the next token, ... 

Then, the ... might be replaced with something indicating why it's important:

> ... so the two are not considered adjacent

Reply:
done. thanks